### PR TITLE
chore: Adding note for migrated repo

### PR DESCRIPTION
**Pull request by ddixit14**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Google Common Protos
 
+🚌 In January 2023, this library has moved to [gapic-generator-java/java-common-protos](https://github.com/googleapis/gapic-generator-java/tree/main//java-common-protos). This repository will be archived in the future. Future releases will appear in the new repository (https://github.com/googleapis/gapic-generator-java/releases).
+The Maven artifact coordinates (com.google.api.grpc:google-common-protos-parent) remain the same.
```
Comment (suztomo) [inline README.md:4]: The users do not use this artifacts. Can you use `grpc-google-common-protos` and `proto-google-common-protos`?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Google Common Protos
 
+🚌 In January 2023, this library has moved to [gapic-generator-java/java-common-protos](https://github.com/googleapis/gapic-generator-java/tree/main//java-common-protos). This repository will be archived in the future. Future releases will appear in the new repository (https://github.com/googleapis/gapic-generator-java/releases).
```
Comment (suztomo) [inline README.md:3]:
```suggestion
🚌 In January 2023, this library has moved to [gapic-generator-java/java-common-protos](https://github.com/googleapis/gapic-generator-java/tree/main/java-common-protos). This repository will be archived in the future. Future releases will appear in the new repository (https://github.com/googleapis/gapic-generator-java/releases).
```

An unnecessary slash.

**Review by suztomo — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # Google Common Protos
 
+🚌 In January 2023, this library has moved to [gapic-generator-java/java-common-protos](https://github.com/googleapis/gapic-generator-java/tree/main/java-common-protos). This repository will be archived in the future. Future releases will appear in the new repository (https://github.com/googleapis/gapic-generator-java/releases).
+The Maven artifact coordinates (com.google.api.grpc:grpc-google-common-protos) and (com.google.api.grpc:proto-google-common-protos) remain the same.
```
Comment (suztomo) [inline README.md:4]:
```suggestion
The Maven artifact coordinates (com.google.api.grpc:grpc-google-common-protos and com.google.api.grpc:proto-google-common-protos) remain the same.
```

The items connected by "and" was strange.

**Review by suztomo — CHANGES_REQUESTED** [review]

**Comment by suztomo** [comment]
@ddixit14 OwlBot reverts change to the templates. Modify owlbot.py at the root to exclude the files.

**Comment by ddixit14** [comment]
Owlbot is not touching these files anymore. Hence merging it in.

**Review by suztomo — APPROVED** [review]

**Pull request merged**